metrics/grafana/tidb.json

```diff
@@ -7912,6 +7912,262 @@
             "alignLevel": null
           }
         },
+        {
```

Comment:
Do we need a panel to differentiate the request durations of internal and external requests? For example the former `distsql` and `transaction` panels  display information about `lock key retry`, `transaction statement number`, `cop duration` all together and it's difficult to tell if the user requests are really slow in these phases.

Reply:
That's a good idea, display the metrics by the internal and external only cover some use cases, metrics by `request_source` have significant amplification. There can be a row containing such 2 types of metrics. I'll move the new added two panels into a new row.